docs/user-guide/activity/README.md

```diff
@@ -54,6 +54,40 @@ You can:
 - You can edit the comment if you are the author  
 - You can quote someone else's comment in a new comment. This is useful if you want to respond to a certain part of a comment.
 
+### Internal comments (Enterprise add-on)
+
+> [!NOTE]
+> Internal comments are an Enterprise add-on and can only be used with [Enterprise cloud](../../enterprise-guide/enterprise-cloud-guide/) or [Enterprise on-premises](../../enterprise-guide/enterprise-cloud-guide/). An upgrade from the free Community edition is easily possible.
+
+You can add work packages with restricited visibility to better manage information flow and share sensitive information on a need to know basis. Comments with restricted visibility are called **internal comments** in OpenProject. You will recognize internal comments by distinctinctly different color scheme and the lock icon. 
```

Comment:
```suggestion
You can add work package comments as internal to better manage information flow and share sensitive information to an authorized group of users. Internal comments are distinguished from other comments via a different color scheme and a lock icon. 
```

Reply:
AFAIK, the term "restricted visibility" was completely replaced with "Internal comments" as the more clear option